pkg/vere/serf.c

```diff
@@ -949,7 +949,7 @@ _serf_writ_live_exit(u3_serf* sef_u, c3_w cod_w)
       fclose(fil_u);
     }
   }
-
+  
```

Comment:
The only change to this file is added whitespace.

Reply:
cleaned up